### PR TITLE
Add vps release

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi dr.max @maximilien,

This PR only contains vps release code(should be small) but it is based on PR #23 (commit c993bad), so please help to merge it firstly. Thank you.

### CODE REVIEW

File: packages/vps/packaging [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	pkg_dir=$(readlink -nf /var/vcap/packages/golang_1.7)
+else
+	pkg_dir=$BOSH_PACKAGES_DIR/golang_1.7
+fi
+
+export GOROOT=$pkg_dir
+export GOPATH=$PWD
+export PATH=$GOROOT/bin:$PATH
+
+mkdir ../src && cp -a * ../src/ && mv ../src ./src
```
Comment (maximilien) [inline packages/vps/packaging]: how do you know you are in correct directory? Seems to me `mkdir ../src ...` is a bit dangerous? Is there a way to guarantee we are in correct start directory?

Look at the [Zookeeper release](https://github.com/cppforlife/zookeeper-release) [`packaging`](zookeeper-release/packages/java/packaging) scripts as examples... but other releases should have that too.

File: packages/vps/spec [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: vps
+
+dependencies:
+  - golang_1.7
+
+files:
+- "**/*"
```
Comment (maximilien) [inline packages/vps/spec]: similar comment here. I think you can be more specific and only copy `golang_1.7` files

**Review by maximilien — CHANGES_REQUESTED** [review]
Please address comments. Also, since I merged previous PR if you resubmit this the change set will be smaller since the BMS code is removed already

File: packages/vps/packaging (hunk 1 above)
Comment (zhanggbj) [reply packages/vps/packaging]: Sure, will make it more specific in next patch.

File: packages/vps/spec (hunk 2 above)
Comment (zhanggbj) [reply packages/vps/spec]: yes, will update in next patch. Thanks.

**Comment by zhanggbj** [comment]
Comments Addressed. Please help to take another review. Thank you.

**Review by maximilien — APPROVED** [review]
Looks great. Merging. Best 👍

**Pull request merged**